src/test/java/org/openstreetmap/atlas/checks/validation/areas/OverlappingAOIPolygonCheckTest.java

```diff
@@ -0,0 +1,57 @@
+package org.openstreetmap.atlas.checks.validation.areas;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.openstreetmap.atlas.checks.configuration.ConfigurationResolver;
+import org.openstreetmap.atlas.checks.validation.verifier.ConsumerBasedExpectedCheckVerifier;
+import org.openstreetmap.atlas.utilities.configuration.Configuration;
+
+/**
+ * Tests for {@link OverlappingAOIPolygonCheck}
+ *
+ * @author bbreithaupt
+ */
+public class OverlappingAOIPolygonCheckTest
```

Comment:
Does this test all possible code paths - overlap and non-overlap, overlapping but not matching filters, overlapping but not having enough overlap?

Reply:
It was missing a test for overlap that is below the minimum. That is now present.